converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/ExceptionStorage/IExceptionStore.cs

```diff
@@ -29,8 +29,9 @@ public interface IExceptionStore
         /// </summary>
         /// <param name="changeFeedEntry">ChangeFeedEntry that threw exception</param>
         /// <param name="retryNum">Number of times the entry has been tried</param>
+        /// <param name="waitTime">Amount of time to wait before retrying</param>
         /// <param name="exceptionToStore">The exception that was thrown and needs to be stored</param>
         /// <param name="cancellationToken">Cancellation token.</param>
-        Task WriteRetryableExceptionAsync(ChangeFeedEntry changeFeedEntry, int retryNum, Exception exceptionToStore, CancellationToken cancellationToken = default);
+        Task WriteRetryableExceptionAsync(ChangeFeedEntry changeFeedEntry, int retryNum, double waitTime, Exception exceptionToStore, CancellationToken cancellationToken = default);
```

Comment:
>waitTime [](start = 96, length = 8)

If it is not a timepan the name should include the units.
Also this should imply its for future wait time. either nextDelayTimeSeconds or futureDelayTimeSeconds

Reply:
updated, thanks